src/main/kotlin/racingcar/domain/RacingGame.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain
+
+import racingcar.domain.CarConfig.randomNumberGenerator
+import racingcar.ui.OutputView
+
+class RacingGame(private val tryCount: Int, private val numberOfCar: Int) {
+
+    private val carCollection = CarCollection(numberOfCar)
+
+    fun runRace() {
+        OutputView.printOutputLetter()
+        for (i in 1..tryCount) {
```

Comment:
```suggestion
        repeat(tryCount)
```
이렇게도 할 수 있을 것 같아요 :)